diff -urN pftp-1.0.15.orig/Makefile pftp-1.0.15/Makefile
--- pftp-1.0.15.orig/Makefile	Mon Jul 26 23:24:40 1999
+++ pftp-1.0.15/Makefile	Mon Jul 26 23:28:56 1999
@@ -29,7 +29,7 @@
 #PREFIX=$(HOME)
 PREFIX=/usr
 BINDIR=$(PREFIX)/bin
-MANDIR=$(PREFIX)/man
+MANDIR=$(PREFIX)/share/man
 
 # Choose your favorite options if you like or let
 # this description file choose its favorite ones.
@@ -39,7 +39,7 @@
 CC=
 
 # If your system has shadow password uncomment
-#HAVE_SHADOW=1
+HAVE_SHADOW=1
 
 # If you do have POSIX threads but don't want to use it comment
 # out the following option.  Otherwise leave it as it is.
@@ -58,7 +58,7 @@
 USE_INET6=yes
 # You will need `-lresolv' on Linux compiled with glibc.
 # Other systems usually don't need `-lresolv'.
-INET6LIBS=-linet6 -lresolv
+INET6LIBS=-lresolv
 
 # If this Makefile couldn't find your IPv6 libraries,
 # please enter the pathes here and uncomment it.
@@ -116,7 +116,7 @@
 ### Linux
 Linux_UNAME=Linux
 Linux_CC=gcc
-Linux_CFLAGS= -O3 -ansi -Wall -fexpensive-optimizations -D_BSD_SOURCE -D_POSIX_SOURCE
+Linux_CFLAGS= -ansi -Wall -fexpensive-optimizations -D_BSD_SOURCE -D_POSIX_SOURCE
 
 ### NEXTSTEP
 NEXTSTEP_UNAME=NEXTSTEP
@@ -391,7 +391,7 @@
 		$$ECHO "Checking IPv6 resources...$$NEWLINE"; \
 		for libdir in $(INET6_LIBPATH) /usr/inet6/lib /usr/local/inet6/lib \
 			$(HOME)/lib; do \
-			if test -f "$$libdir/libinet6.a"; then \
+			if test -f "/usr/include/netinet/ip6.h"; then \
 				INET6="-DHAVE_INET6"; \
 				LIBINET6="-L$$libdir $(INET6LIBS)"; \
 			fi; \
